samples/Rakefile

```diff
@@ -1,102 +1,58 @@
 #
-#  Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #  SPDX-License-Identifier: Apache-2.0
 
-require 'open-uri'
+require "open-uri"
+require "rexml/document"
 
 SAMPLES_DIR = File.dirname(__FILE__)
-JAR_DIR = File.join(SAMPLES_DIR, 'jars')
```

Comment:
What's the difference of using single quotes and double quotes?

Reply:
The ruby formatter modifies all the single quotes to be consistent 😂